### DISCUSSION TITLE
Key points from ENS RFP for streaming services

### DISCUSSION THREAD

**Discussion by smol-ninja** [sablier-labs/flow#18] [Ideas]
From: https://discuss.ens.domains/t/rfp-stream-platform-for-ens-service-providers/18448

> Are you able to create indefinite streams? How easy is it to renew, pause or edit an existing stream?
Our current stream providers will be receiving funds for at least one year, but there’s no end date. The goal is to have a new Stream Provider election in Q1 or Q2 2025, followed by an Executable Proposal which will remove some recipients, add others and maybe change the value of some. Your stream must be able to support that without any interruption in the service.

Looks like a requirement for EOES as there is no end date.

Key points from this:
1. ability to remove some recipients -> equivalent to closing stream
2. add new recipients -> equivalent to opening new streams
3. change the values -> equivalent to closing followed by opening a new stream. Or should we allow updating recipients as well as rps for an existing stream? Thoughts??

**Comment by IaroslavMazur** [comment]
> should we allow updating recipients **as well as rps** for an existing stream?

You must've meant "alongside rps", because the rate per second can already be modified via [SablierV2OpenEnded.adjustRatePerSecond()](https://github.com/sablier-labs/v2-open-ended/blob/cbdd7905d79e43252724895564553d75fdba40d7/src/SablierV2OpenEnded.sol#L128)

In any case, we don't stand a chance with this RFP, because of this requirement of theirs:
> Platforms must be on main net ethereum **for over at least one year** and have a streamed **over $100 million in total value streamed**

Besides, there's, probably, no chance EOES will be ready to launch anywhere soon enough for ENS to consider it, anyways, given that they're planning on opening the Streams "by mid-january to February"...

**Reply by smol-ninja** [reply]
> You must've meant "alongside rps"

Yes thanks for correcting @IaroslavMazur, my bad.

> we don't stand a chance with this RFP

Oh! I thought they meant protocol in general. But you are right.

**Comment by andreivladbrg** [comment]
Can we close this @smol-ninja ?

**Reply by smol-ninja** [reply]
Yes.
